src/test/java/roomescape/reservation/repository/ReservationDaoImplTest.java

```diff
@@ -0,0 +1,105 @@
+package roomescape.reservation.repository;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.JdbcTest;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.test.annotation.DirtiesContext;
+import roomescape.reservation.entity.Reservation;
+import roomescape.reservation.entity.ReservationTime;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+@JdbcTest
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+class ReservationDaoImplTest {
+
+    @Autowired
+    NamedParameterJdbcTemplate jdbcTemplate;
+    private ReservationDaoImpl reservationDao;
```

Comment:
private 선언 + 가독성을 위해 줄바꿈도 추가하면 좋을 것 같네요 😄 

```suggestion
    @Autowired
    private NamedParameterJdbcTemplate jdbcTemplate;
    
    private ReservationDaoImpl reservationDao;
```

Reply:
수정하였습니다!